tests/__init__.py

```diff
@@ -15,6 +15,7 @@ class Test(object):
         "configure": ["configure.sh"],
         "make": ["make.sh", "cmake.sh"],
         "transpile": ["transpile.gen.sh", "transpile.sh"],
+        "refactor": ["refactor.gen.sh", "refactor.sh"],
```

Comment:
Do we still want to run tests against the non-refactored code?  The transpiler still needs to work on its own.